.php_cs.dist

```diff
@@ -1,12 +1,16 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
Can you confirm this line has nothing to do with `native_function_invocation`? It's not a problem you can let it in the PR, I just want to be sure 🙂

Reply:
Yes, it has nothing to do with `native_function_invocation`, it's a way to tell PHP to enforce types.